readme.adoc

```diff
@@ -235,3 +235,5 @@ jvm_gc_collection_seconds_count{gc="PS MarkSweep",} 6.0
 jvm_gc_collection_seconds_sum{gc="PS MarkSweep",} 0.565
 ```
 
+== OpenTracing/Jaeger
+The application traces all requests to Jaeger (https://uber.github.io/jaeger/) and the result can be viewed on http://127.0.0.1:8090
```

Comment:
Add what would a sample output look like?

Reply:
Will add a small screenshot of the Jaeger UI with some tracing examples